omod/src/main/webapp/fragments/summary/section.gsp

```diff
@@ -29,7 +29,7 @@
             <div>
                 <h3>${ ui.message(nameTemplate.nameMappings[name]) } </h3>
                 <p class="left">
-                    ${ ui.message(initialNameFieldValue) }
+                    ${ ui.message(ui.encodeHtml(initialNameFieldValue)) }
```

Comment:
I understand `ui.encodeHtml(String)` works out of the box for the both html attributes and html text content but I believe `ui.encodeHtmlContent(String)` is more optimised for the text content? It's just a concern; not such a serious requirement.